executor/prepared.go

```diff
@@ -160,11 +161,16 @@ func (e *PrepareExec) DoPrepare() {
 	sorter := &paramMarkerSorter{markers: extractor.markers}
 	sort.Sort(sorter)
 	e.ParamCount = len(sorter.markers)
+	for i := 0; i < e.ParamCount; i++ {
+		sorter.markers[i].Order = i
+	}
 	prepared := &Prepared{
 		Stmt:          stmt,
 		Params:        sorter.markers,
 		SchemaVersion: e.IS.SchemaMetaVersion(),
 	}
+	inPrepare := true
+	prepared.UseCache = plan.Cacheable(stmt, inPrepare)
```

Comment:
just `prepared.UseCache = plan.Cacheable(stmt, true)`

Reply:
Thank you for the suggestion. I've changed the codes as follows:

`-  inPrepare := true`
`-  prepared.UseCache = plan.Cacheable(stmt, inPrepare)`
`+  prepared.UseCache = plan.Cacheable(stmt, true)`
` `